### ISSUE TITLE
Why is InterIntegratedCircuit missing on NuGet?

### ISSUE THREAD

**Issue by MihaMarkic** [raspberry-sharp/raspberry-sharp-io#62]
Why is InterIntegratedCircuit missing on NuGet?

**Comment by Kefner** [comment]
bump

**Comment by xlevel** [comment]
Same here. Is this something that someone could take on for the project?

**Comment by JTrotta** [comment]
You were right. Next week I will fork the project with some corrections under debugging, and port it on Nuget. Please wait...

**Comment by paultechguy** [comment]
bump.  Any chance of the updated nuget with InterIntegratedCircuit and Raspberry.IO.Components.Controllers.Pca9685?

**Comment by JerseyTechGuy** [comment]
It's now April 2017 and InterIntegratedCircuit is still missing.  Might be time to remove I2C from the description. Very misleading.

**Comment by guehmann** [comment]
Today wanted to include the SPI Assembly in my project via nuget, but Raspberry.IO.SerialPeripheralInterface is also missing. Using Mono on Ubuntu but it's the same on Windows with Visual Studio. Do I'm something wrong? So i cloned the project and build it with Mono, build succeeded.

**Comment by paultechguy** [comment]
Yes, had same issue so once I built it, I built my own nuget (local) and
use that with all the needed assemblies.

**Comment by guehmann** [comment]
@paultechguy perhaps you have an example of a working implementation of MIfare RC522 with a SPI?
Thanks in advance.
Jens

**Comment by paultechguy** [comment]
@guehmann, sorry, do not have that done.  I ended up packing the sharpio libs with my app...all the ones I needed.

**GPIO Examples in C#**
https://github.com/paultechguy/RaspberryPi.DotNet.GpioExamples

**Extensible GPIO Web Services in C#**
https://github.com/paultechguy/RaspberryPi.DotNet.GpioWeb

**Comment by guehmann** [comment]
@paultechguy thank you very much, that's great. think will help me.
Regards Jens.
